plot.sh

```diff
@@ -112,20 +115,27 @@ create_summary_plot () {
   plot_directory=$3
   snapshot_name=$4
   catalogue_name=$5
+  ptype=$6
 
   output_path=$plot_directory/$run_name
 
   old_directory=$(pwd)
   cd $output_path
 
+  if [ "$ptype" = "COLIBRE" ]; then
+        tstring="colibre"
```

Comment:
Can we pass the lower case version of this and then we don't need to have this check?